signature_gen/src/main.rs

```diff
@@ -88,6 +88,67 @@ fn generate_2048_bit_signature_parameters(msg: &str, as_toml: bool, exponent: u3
     }
 }
 
+
+fn generate_1024_bit_signature_parameters(msg: &str, as_toml: bool, exponent: u32) {
```

Comment:
Unless I am missing something this looks like it could be combined with `generate_2048_bit_signature_parameters` and we accept another `bits` param. I won't block on this though and it can be done in a follow-up

Reply:
👍 indeed it could - it was just a workaround on my part to generate test vectors for the 1024 bit test